frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts

```diff
@@ -3,11 +3,13 @@ import _ from "underscore";
 
 import validate from "metabase/lib/validate";
 import { humanize, slugify } from "metabase/lib/formatting";
+import { moveElement } from "metabase/visualizations/lib/utils";
```

Comment:
Should we move `moveElement` to a more generic place like `metabase/lib/something`?